manifests/params.pp

```diff
@@ -19,6 +19,12 @@
           $debug_level      = '0'
           $randomwait       = '60'
         }
+        
+        '2': {
```

Comment:
You could add this to the same line as the case for 7 (like this - `'7', '2':`)

Reply:
I have added your suggestion. Thanks!